benchmarks/QueryToLatexBench.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mathematicator\Tokenizer\Benchmarks;
+
+use Mathematicator\Tokenizer\Tokenizer;
+use Nette\Configurator;
+
+if (\is_file($autoload = __DIR__ . '/../vendor/autoload.php')) {
+	require_once $autoload;
+}
+
+/**
+ * @BeforeMethods({"init"})
+ */
+class QueryToLatexBench
+{
+
+	/** @var Tokenizer */
+	private $tokenizer;
+
+
+	public function init()
```

Comment:
In future commit please use `void` for methods without `return` statement.

```suggestion
	public function init(): void
```